Microsoft.DotNet.Interactive/LanguageService/MarkupKind.cs

```diff
@@ -1,13 +1,8 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Newtonsoft.Json;
-using Newtonsoft.Json.Converters;
-using Newtonsoft.Json.Serialization;
-
 namespace Microsoft.DotNet.Interactive.LanguageService
 {
-    [JsonConverter(typeof(StringEnumConverter), converterParameters: typeof(CamelCaseNamingStrategy))]
     public enum MarkupKind
```

Comment:
>MarkupKind [](start = 16, length = 10)

This doesn't belong at the domain layer, as it seems like an artifact of LSP. We should favor behaviors/types over enums here.

Reply:
What about `abstract class ContentBase` and `class PlainTextContent : ContentBase`/`class MarkdownContent : ContentBase`?  I'm open to naming suggestions if you don't like those.  Then the `MarkupKind` enum will exist only at the LSP serialization layer.